src/System.Diagnostics.Process/src/System/Diagnostics/ProcessStartInfo.Windows.cs

```diff
@@ -37,5 +37,19 @@ public bool LoadUserProfile
             get { return _loadUserProfile; }
             set { _loadUserProfile = value; }
         }
+
+        // CoreCLR can't correctly support UseShellExecute=true for the following reasons
+        // 1. ShellExecuteEx is not supported on onecore.
```

Comment:
That's a question for @pallavit.  @nguerrera asked a similar question at https://github.com/dotnet/corefx/issues/4794.  For this PR, I've simply moved this existing code from one file to another.